extensions/amp-autocomplete/0.1/test/test-amp-autocomplete-init.js

```diff
@@ -77,6 +78,25 @@ describes.realWin(
       return ampAutocomplete.build().then(() => ampAutocomplete);
     }
 
+    it('should require experiment when "query" is specified', () => {
+      return allowConsoleError(() => {
+        return expect(getAutocomplete({'query': 'q'})).to.be.rejectedWith(
+          /Experiment amp-autocomplete is not turned on/
+        );
+      });
+    });
+
+    it('should build with "inline" and "query" when specified with experiment on', () => {
```

Comment:
1. Can we flatten promise chains? Tests can use `async/await`.
2. Can we toggle the experiments using `describes` config?
3. nit: you don't need string quotes for keys in tests

```js
it('should build with "inline" and "query"...', async () => {
  const {implementation_} = await getAutocomplete({filter: 'substring', query': 'q'});
  expect(implementation_.queryKey_).to.equal('q');
});
```

Reply:
I am happy to address 1/2/3 generally but want to ask: Since making the change would require _either_ causing this one-off test to be inconsistent with the rest of the tests _or_ changing the majority if not all of this test file, what do you think about addressing in a follow up PR?

Reply:
Follow-up is fine, but it's ok to create new tests in a more modern way even if they're inconsistent with the existing ones.